Formula/proxysql.rb

```diff
@@ -0,0 +1,37 @@
+class Proxysql < Formula
+  desc "High Performance Advanced Proxy for MySQL"
+  homepage "http://www.proxysql.com/"
+  url "https://github.com/sysown/proxysql/archive/v1.4.10.tar.gz"
+  sha256 "29020cba75b778ef45b06c5469ee5930ce80afbd0290a112d8c2a6652b4c8a2e"
+
+  # Build dependencies listed here: https://github.com/sysown/proxysql/blob/master/INSTALL.md
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "mysql@5.6" => :test
+  depends_on "openssl"
+
+  def install
+    system "make"
+    # Currently, the Proxysql build system uses fixed paths for install locations.
+    # See https://github.com/sysown/proxysql/blob/master/Makefile
+    # The 'make install' target would need to be patched to support Homebrew.
+    # Configurable target locations are requested in https://github.com/sysown/proxysql/issues/1565
```

Comment:
We might well want to wait for upstream to do something about this, given the issue seems to have been ignored thus far. Would prefer not to get stuck with manual `.install` calls indefinitely.

Reply:
Unfortunately the upstream is in an unfortunate spot of having a) not many maintainers, b) a very opinionated build system, and c) the assumption about bundled libraries and fixed paths baked into the build process at multiple points.

I'd love to see it change, but I do not consider it likely, especially given that Renée &co have a ton of other stuff they're working on feature-wise to get 2.0.0 released.

On the bright side, these manual `install` calls are a bit less sketchy than they could be, since the proxysql app compiles in all of its dependencies, and tries very hard to make its deployable a single filesystem-location-independent binary. As a result, I think this is more of the "ugly" variety of `install` hacks and  less of the "potential breakage" variety. What do you think?